.github/workflows/ci_cd.yml

```diff
@@ -278,6 +278,80 @@ jobs:
           include_passed: true
           fail_on_failure: true
 
+  embedding-scripts-tests:
+    name: Embedding scripts testing and coverage
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    needs: [style, revn-variations]
```

Comment:
```suggestion
    needs: [smoke-tests, revn-variations]
```

Reply:
I added this changes in last PR where we do smoke tests first which takes style. Then after smoke tests we do all embedding, remote tests.